docs/Tutorials/zot-registry-tutorial.mdx

```diff
@@ -0,0 +1,236 @@
+---
+title: Tutorial distribute an OCI artifact to Zot Registry using ORAS CLI
+sidebar_position: 0
+---
+
+# Tutorial to Install Zot Registry and Distribute OCI Artifacts to Zot Registry using ORAS CLI
+
+Before we tackle how to distribute an OCI artifact to an OCI registry, let us first take a look at what we mean by distributing OCI artifacts, why we need it and what OCI registries are. 
+
+OCI registries are like a single location where container images are stored and made available to everyone. Distributing OCI artifacts refers to when artifacts are pushed to these registries. These artifacts can then be pulled (or retrieved) by someone else and used in their environments. These images often help with packaging and distributing software applications.
+
+We will be using zot registry in this guide. Zot registry is an OCI-native container registry for distributing container images and OCI artifacts.
```

Comment:
Is that what they say OCI-native or do they say OCI compliant?

Reply:
It's OCI-native on their website that's why I've used that term.

Reply:
because ... 

zot: a production-ready vendor-neutral OCI image registry - images stored in [OCI image format](https://github.com/opencontainers/image-spec), [distribution specification](https://github.com/opencontainers/distribution-spec) on-the-wire, that's it!


That said, it also supports oras artifacts/referrers until OCI artifacts land (undecided at this time though)

Reply:
Thank you @rchincha!